root/documentation/vep.conf

```
@@ -194,6 +194,11 @@
         description=LOFTEE identifies LoF (loss-of-function) variation. See <a target="_blank" href="https://github.com/konradjk/loftee/blob/master/README.md">README</a> for more details.
         default=0
       </LoF>
+      <UTRannotator>
```

The name of the plugin needs to be fixed here and in all other instances where being used.

```suggestion
      <UTRAnnotator>
```

---

Done.